org.pegasus_frontend.Pegasus.yml

```diff
@@ -1,6 +1,6 @@
 app-id: org.pegasus_frontend.Pegasus
 runtime: org.kde.Platform
-runtime-version: '5.15-21.08'
```

Comment:
Has been done by `org.flathub.flatpak-external-data-checker` when I ran it locally.